res/values-hr/strings.xml

```diff
@@ -597,8 +597,8 @@
     <string name="PRIZES">"NAGRADE"</string>
 
     <string name="Enable_Custom_Skin">"Omogući Prilagođene Skinove"</string>
-    <string name="Use_Custom_Skin">"Prilagođeni\nskin"</string>
-    <string name="custom_skin_description">"Ova značajka omogućuje vam da koristite svoj profil sliku ili datoteku kao svoj skin. Ova postavka će samo promjeniti skin koji vi vidite. Drugi ljudi neće vidjeti vaš vlastiti skin."</string>
+    <string name="Use_Custom_Skin">"Koristiti Prilagođenu Kožu"</string>
```

Comment:
Please return the `\n` between "Prilagođenu" and "Kozu".

Reply:
> Molimo vratite `\n`između "Koristiti" i "Prilagođenu"; "Kožu" je prebukvalno, pa ga zadržite kao "Skin".

Promjene su poslate